scripts/docs_config/glossary.rst

```diff
@@ -0,0 +1,97 @@
+Glossary
+==========================================================
+
+Homogeneous Memory
+  A collection of memory composed of a single memory type, managed by a singular 
+  driver using a uniform approach.
+
+Heterogeneous Memory
+  A set of memory composed of multiple types of memory technologies, each 
+  requiring distinct handling approaches often managed by separate drivers.
+
+Memory Tiering
+  An organization and hierarchy of different types of memory storage within a 
+  system, with each type of memory having distinct characteristics, performance, 
+  and cost attributes. These memory tiers are typically organized in a 
+  hierarchy, with faster, more expensive memory located closer to the processor 
+  and slower, less expensive memory located further away.
+
+Memory Access Initiator 
+  A component in a computer system that initiates or requests access to the 
+  computer's memory subsystem. This could be a CPU, GPU, or other I/O and cache 
+  devices.
+
+Memory Target 
+  Any part of the memory subsystem that can handle memory access requests. This 
+  could be the OS memory (RAM), video memory that resides on the graphics 
```

Comment:
I would change 'OS memory' to 'main memory' or something like that.

Reply:
done